qiskit/transpiler/target.py

```diff
@@ -1263,6 +1269,7 @@ def from_configuration(
             instruction_durations: Optional instruction durations for instructions. If specified
                 it will take priority for setting the ``duration`` field in the
                 :class:`~InstructionProperties` objects for the instructions in the target.
+            meas_map: List of sets of qubits that must be measured together.
```

Comment:
Again, only one of the two accepted formats is documented here.

Reply:
As you pointed out, the document is not easy for users or developers to understand.
I fixed at https://github.com/Qiskit/qiskit-terra/pull/10258/commits/407f87d192ef1b76683ddf9ff81d2b9b4034d47e .